src/test/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSinkUnitTest.java

```diff
@@ -27,9 +31,20 @@
 import java.util.List;
 
 public class ReadsSparkSinkUnitTest extends BaseTest {
+    private MiniDFSCluster cluster;
```

Comment:
It is to costly.  Spinning up / down the cluster is ~10s.  Way to slow for every method.  I think we'd actually do better to have a single cluster used for all the tests, and enforce uniqueness with file names like we do on the local file system.  This seemed like a happy medium between them since I figured people wouldn't like that though.